dipy/viz/window.py

```diff
@@ -486,6 +469,52 @@ def start(self):
         del self.iren
         del self.window
 
+    def record(self, filename="record.log"):
```

Comment:
Let's change the name to record_events (because we are already using this name in another ) and the next one play_recorded_events. Or we could add parameters to start(mode=None or 'record' or 'play')